<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="method_list">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Шифр Цезаря</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Шифр Виженера</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RSA шифрование</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Двоичный код</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="mode">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Режим: Шифрование</string>
    </property>
   </widget>
   <widget class="QPushButton" name="do_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>160</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>DO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="output_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Входящий текст</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Выходящий текст</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="keys">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Доп. параметры</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ru_radio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Русский</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="en_radio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Английский</string>
    </property>
   </widget>
   <widget class="QPushButton" name="create_key_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать ключи</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
